### PR TITLE
OSDOCS-12604: inline containerfiles and update with dnf

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.17+

Issue: https://issues.redhat.com/browse/OSDOCS-12604

Link to docs preview: https://84641--ocpdocs-pr.netlify.app/openshift-enterprise/latest/machine_configuration/mco-coreos-layering.html#coreos-layering-examples_mco-coreos-layering

QE review:
- [x] QE has approved this change.

Additional information: related to #81649

### CODE REVIEW

**Comment by jldohmann** [comment]
@jlebon @HuijingHei ive updated the coreos layering containerfiles to not be pulled from that older repo 🙂 and for them to use `dnf` could you PTAL? thank you!

File: machine_configuration/mco-coreos-layering.adoc [hunk 1]
```diff
@@ -115,7 +127,15 @@ RUN rpm-ostree install https://dl.fedoraproject.org/pub/epel/epel-release-latest
 .Example out-of-cluster Containerfile to apply a third-party package from EPEL
 [source,yaml]
 ----
-include::https://raw.githubusercontent.com/openshift/rhcos-image-layering-examples/master/htop/Containerfile[]
+# Get {op-system} base image of target cluster `oc adm release info --image-for rhel-coreos`
+# hadolint ignore=DL3006
+FROM quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256...
+
+#Enable EPEL (more info at https://docs.fedoraproject.org/en-US/epel/ ) and install htop
+RUN rpm-ostree install https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm && \
```
Comment (jlebon) [inline machine_configuration/mco-coreos-layering.adoc:135]:
```suggestion
RUN dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm && \
```

**Review by jlebon — APPROVED** [review]
Thanks, LGTM!

File: machine_configuration/mco-coreos-layering.adoc (hunk 1 above)
Comment (jldohmann) [reply machine_configuration/mco-coreos-layering.adoc:135]: ah, ty

**Review by HuijingHei — APPROVED** [review]

**Review by xenolinux — COMMENTED** [review]
LGTM

**Comment by jldohmann** [comment]
i just noticed attributes arent resolving in the code block, so subsequent pushes are to fix that

**Comment by jldohmann** [comment]
will need another PR to inline these examples for 4.12 through 4.16 without `dnf`

**Comment by jlebon** [comment]
> will need another PR to inline these examples for 4.12 through 4.16 without `dnf`

FWIW, I don't think this is strictly necessary. The repo itself isn't going away, and I think having it be read-only shouldn't break the openshift-docs sourcing from there.

**Pull request merged**